studioShared/lib/interfaces/customerCases.ts

```diff
@@ -0,0 +1,15 @@
+import { PortableTextBlock } from "sanity";
+
+import { Slug } from "./global";
```

Comment:
`Slug` can be imported from `sanity` instead of defining a custom type (just like `PortableTextBlock`).